src/smart-components/ChannelList/utils.js

```diff
@@ -248,7 +248,9 @@ export const pubSubHandler = (pubSub, channelListDispatcher) => {
   subScriber.set(topics.UPDATE_USER_MESSAGE, pubSub.subscribe(topics.UPDATE_USER_MESSAGE, (msg) => {
     const { channel, message } = msg;
     const updatedChannel = channel;
-    updatedChannel.lastMessage = message;
+    if (updatedChannel.lastMessage && updatedChannel.lastMessage.messageId === message.messageId) {
```

Comment:
updatedChannel?.lastMessage?.messageId

Reply:
The type of this file is not typescript

Reply:
its okay, works in js. Is JS feature

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Oh, I didn't know that. Thanks